src/pages/listing.js

```diff
@@ -116,7 +116,7 @@ export default function Listings() {
             overallRating,
             costPerNight,
             bookings
-          } = data?.listing;
+          } = data.listing;
```

Comment:
was there a particular reason to remove the `?.`?

Reply:
Yeah it was resulting in an error and wouldn't run the app! Have a feeling it was related to upgrading react-scripts.